_data/projects/Trapheus.yml

```
@@ -0,0 +1,13 @@
+name: Trapheus
+desc: Trapheus automates restoration of RDS database instances from snapshots into any dev, staging or production environments.It supports individual RDS Snapshot as well as cluster snapshot restore operations.
+site: https://github.com/intuit/Trapheus
+tags:
+  - rds
+  - step-functions
+  - python
+  - aws-lambda
+  - aws-lambda-layers
+  - database-restore
+upforgrabs:
+  name: up for grabs
```

I don't see this label being used in your project - should this be changed to `good first issue`?

```suggestion
  name: good first issue
```